### PR TITLE
Handling for default container log max size

### PR DESCRIPTION
**Pull request by UtheMan**
Set default value for Container Log Max Size to 50MB

### CODE REVIEW

File: pkg/templates/templates_generated.go [hunk 1]
```diff
@@ -4568,6 +4623,43 @@ func linuxCloudInitArtifactsUbuntuCse_install_ubuntuSh() (*asset, error) {
 	return a, nil
 }
 
+var _linuxCloudInitConfigHcpmumanskiebld51438502EastusSvc0 = []byte(`apiVersion: v1
```
Comment (anujmaheshwari1) [inline pkg/templates/templates_generated.go:4626]: some make target probably messed up depending on which you ran? these shouldnt be here

File: pkg/templates/templates_generated.go [hunk 2]
```diff
@@ -58,6 +61,7 @@
 // linux/cloud-init/artifacts/sysctl-d-60-CIS.conf
 // linux/cloud-init/artifacts/ubuntu/cse_helpers_ubuntu.sh
 // linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh
+// linux/cloud-init/config.hcpmumanskiebld51438502-eastus-svc-0
```
Comment (anujmaheshwari1) [inline pkg/templates/templates_generated.go:64]: aah this is the culprit I suppose. and L3-5.

File: pkg/agent/utils_test.go [hunk 3]
```diff
@@ -258,3 +417,11 @@ var _ = Describe("Assert ParseCSE", func() {
 		Expect(err.Error()).To(ContainSubstring(testMessage))
 	})
 })
+
+func TestKubeletConfigDefaultsShouldBeSetIfCustomKubeletConfigNotProvided(t *testing.T) {
```
Comment (anujmaheshwari1) [inline pkg/agent/utils_test.go:421]: why do we have empty tests?

File: pkg/templates/templates_generated.go (hunk 1 above)
Comment (ganeshkumar5699) [reply pkg/templates/templates_generated.go:4626]: re-running `make generate` again with a clean branch with only necessary changes, may make it work or at least help check that this was added by the make target.

Comment (UtheMan) [reply pkg/templates/templates_generated.go]: Seems like removing accidentally added files helped with make generate

**Review by alexeldeib — APPROVED** [review]

**Review by ganeshkumar5699 — APPROVED** [review]

**Pull request merged**